src/octoprint/plugins/eventmanager/__init__.py

```diff
@@ -19,9 +19,11 @@ def on_settings_load(self):
         for event in events.get("subscriptions", []):
             if "name" not in event:
                 event["name"] = event["command"]
+            if not type(event["event"]) == list:
```

Comment:
```suggestion
            if not isinstance(event["event"], list):
```

(otherwise a sub class of list would cause issues here)